setup.py

```diff
@@ -30,6 +30,7 @@
 
 
 requirements = [
+    'elasticsearch>=2.2.0.dev0,<3.0.0',
```

Comment:
So I realized its convenient as it checks that the correct version is installed when running the site. E.g. someone pulls master and then `inveniomanage` will break until they run `pip install -r requirements.txt`. Without it, things may "seem" to work, but people may be running outdated stuff.

I can add a note about this in the commit message.